### PR TITLE
Add myself to v3 javascript/maintainers.md

### PR DESCRIPTION
**Pull request by Calamari**

### CODE REVIEW

File: languages/javascript/maintainers.md [hunk 1]
```diff
@@ -8,4 +8,4 @@ TODO: add senior maintainers
 
 ## Contributing Maintainers
 
-TODO: add contributing maintainers
+- Georg Tavonius (Slack: @Georg Tavonius, GitHub: Calamari)
```
Comment (tehsphinx) [inline languages/javascript/maintainers.md]:
```suggestion
- Georg Tavonius (Slack: @Georg Tavonius, GitHub: [Calamari](https://github.com/calamari))
```

Comment (tehsphinx) [reply languages/javascript/maintainers.md]: Maybe a link is nicer 😄

Comment (Calamari) [reply languages/javascript/maintainers.md]: Sure, I copied the style from another file, but does not mean we cannot improve it

**Comment by tehsphinx** [comment]
All maintainers were asked to do this...

**Comment by SleeplessByte** [comment]
@tehsphinx yeah, @iHiD just informed us :)

**Pull request merged**